src/client/client.ts

```diff
@@ -203,6 +326,139 @@ export default class CTProtoClient<AuthRequestPayload, AuthResponsePayload, ApiR
     });
   }
 
+  /**
+   * This method sends one chunk of the uploading file
+   *
+   * @param file - sending file
+   * @param chunkNumber - number of sending chunk
+   * @param message - additional info for chunk
+   */
+  private sendChunk(file: FileToUpload<ApiResponse['payload']>, chunkNumber: number, message: string): void {
+
+    /**
+     * Getting chunk by slicing file by the chunk number and buffer limit
+     */
+    const chunkOffset = chunkNumber * this.bufferLimit;
+    const dataOfFile = file.fileData.slice( chunkOffset, chunkOffset + this.bufferLimit );
+    const chunk = Buffer.alloc(this.bufferLimit, 0);
+
+    dataOfFile.copy(chunk, 0);
+    /**
+     * Getting info converted to binary type, which includes info about chunk number and chunk size
+     */
+    const chunkInfo = this.makeDataAboutChunk(chunkNumber, chunk.length);
```

Comment:
this method still not clear. What is a `metaData` here? 

<img width="711" alt="image" src="https://user-images.githubusercontent.com/3684889/170183906-e1242d97-40dc-4f0f-abe0-f6ecb9ba8f44.png">